### DISCUSSION TITLE
How to remove notes/tickables/SVGGElements (Memory Leak)?

### DISCUSSION THREAD

**Discussion by avisilber88** [0xfe/vexflow#1408] [Q&A]
Hi! 
I've spent the better part of the last week trying to troubleshoot my midi sheet music practice game I'm making for my students. The memory heap keeps piling up with SVGGElements that it collects from each of my tickables and/or context.opengroup(). I found that when I delete the <g> nodes using a for-each method, they just become detached SVGGElements. So I figure I have to remove them the right way... but I cannot find a right way in any of the documentation. Please help!!
Best,
Avi

**Comment by FRutkowski** [comment]
I have the same problem
